lib/automation/report_runner.rb

```diff
@@ -138,7 +149,22 @@ def run!
 
             #{input}
           RAW
-          PostCreator.create!(@sender, raw: raw, topic_id: post.topic_id, skip_validations: true)
+            PostCreator.create!(@sender, raw: raw, topic_id: post.topic_id, skip_validations: true)
+          end
+        end
+
+        if @email_receivers.present?
+          @email_receivers.each do |to_address|
```

Comment:
`@email_receivers` should always be an array, so we technically don't need the `#present?` check since `#each` would be a no-op on an empty array, but maybe there's a reason for us to do the check. 🙂

Reply:
I like to keep this nil safe just in case ... but yeah